proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -240,6 +240,11 @@ type executionPayloadJSON struct {
 	Transactions  []hexutil.Bytes `json:"transactions"`
 }
 
+type getPayloadV2ResponseJson struct {
+	executionPayload *executionPayloadCapellaJSON `json:"executionPayload"`
+	blockValue       string                       `json:"blockValue"`
+}
```

Comment:
You added json tags to unexported fields. If I recall correctly the JSON package only works on exported ones.